include/z64player.h

```diff
@@ -7,50 +7,139 @@
+    /* 0x00 */ PLAYER_MASK_NONE,
+    /* 0x01 */ PLAYER_MASK_TRUTH,
+    /* 0x02 */ PLAYER_MASK_KAFEI,
+    /* 0x03 */ PLAYER_MASK_ALL_NIGHT,
+    /* 0x04 */ PLAYER_MASK_BUNNY,
+    /* 0x05 */ PLAYER_MASK_KEATON,
+    /* 0x06 */ PLAYER_MASK_GARO,
+    /* 0x07 */ PLAYER_MASK_ROMANI,
+    /* 0x08 */ PLAYER_MASK_CIRCUS_LEADER,
+    /* 0x09 */ PLAYER_MASK_POSTMAN,
+    /* 0x0A */ PLAYER_MASK_COUPLE,
+    /* 0x0B */ PLAYER_MASK_GREAT_FAIRY,
+    /* 0x0C */ PLAYER_MASK_GIBDO,
+    /* 0x0D */ PLAYER_MASK_DON_GERO,
+    /* 0x0E */ PLAYER_MASK_KAMARO,
+    /* 0x0F */ PLAYER_MASK_CAPTAIN,
+    /* 0x10 */ PLAYER_MASK_STONE,
+    /* 0x11 */ PLAYER_MASK_BREMEN,
+    /* 0x12 */ PLAYER_MASK_BLAST,
+    /* 0x13 */ PLAYER_MASK_SCENT,
+    /* 0x14 */ PLAYER_MASK_GIANT,
+    /* 0x15 */ PLAYER_MASK_FIERCE_DEITY,
+    /* 0x16 */ PLAYER_MASK_GORON,
+    /* 0x17 */ PLAYER_MASK_ZORA,
+    /* 0x18 */ PLAYER_MASK_DEKU,
+    /* 0x19 */ PLAYER_MASK_MAX
 } PlayerMask;
 
+typedef enum {
+    /* 0x00 */ PLAYER_AP_NONE,
+    /* 0x01 */ PLAYER_AP_UNK_1,
+    /* 0x02 */ PLAYER_AP_UNK_2,
+    /* 0x03 */ PLAYER_AP_SWORD_KOKIRI,
+    /* 0x04 */ PLAYER_AP_SWORD_RAZOR,
+    /* 0x05 */ PLAYER_AP_SWORD_GILDED,
+    /* 0x06 */ PLAYER_AP_SWORD_GREAT_FAIRY,
+    /* 0x07 */ PLAYER_AP_STICK,
+    /* 0x08 */ PLAYER_AP_UNK_8,
+    /* 0x09 */ PLAYER_AP_BOW,
+    /* 0x0A */ PLAYER_AP_BOW_FIRE,
+    /* 0x0B */ PLAYER_AP_BOW_ICE,
+    /* 0x0C */ PLAYER_AP_BOW_LIGHT,
+    /* 0x0D */ PLAYER_AP_HOOKSHOT,
+    /* 0x0E */ PLAYER_AP_BOMB,
+    /* 0x0F */ PLAYER_AP_POWDER_KEG,
+    /* 0x10 */ PLAYER_AP_BOMBCHU,
+    /* 0x12 */ PLAYER_AP_NUT = 0x12,
+    /* 0x13 */ PLAYER_AP_PICTO_BOX,
+    /* 0x14 */ PLAYER_AP_OCARINA,
+    /* 0x15 */ PLAYER_AP_BOTTLE,
+    /* 0x16 */ PLAYER_AP_BOTTLE_FISH,
+    /* 0x17 */ PLAYER_AP_BOTTLE_SPRING_WATER,
+    /* 0x18 */ PLAYER_AP_BOTTLE_HOT_SPRING_WATER,
+    /* 0x19 */ PLAYER_AP_BOTTLE_ZORA_EGG,
+    /* 0x1A */ PLAYER_AP_BOTTLE_DEKU_PRINCESS,
+    /* 0x1B */ PLAYER_AP_BOTTLE_GOLD_DUST,
+    /* 0x1D */ PLAYER_AP_BOTTLE_SEA_HORSE = 0x1D,
+    /* 0x1E */ PLAYER_AP_BOTTLE_MUSHROOM,
+    /* 0x1F */ PLAYER_AP_BOTTLE_HYLIAN_LOACH,
+    /* 0x20 */ PLAYER_AP_BOTTLE_BUG,
+    /* 0x21 */ PLAYER_AP_BOTTLE_POE,
+    /* 0x22 */ PLAYER_AP_BOTTLE_BIG_POE,
+    /* 0x23 */ PLAYER_AP_BOTTLE_POTION_RED,
+    /* 0x24 */ PLAYER_AP_BOTTLE_POTION_BLUE,
+    /* 0x25 */ PLAYER_AP_BOTTLE_POTION_GREEN,
+    /* 0x26 */ PLAYER_AP_BOTTLE_MILK,
+    /* 0x27 */ PLAYER_AP_BOTTLE_MILK_HALF,
+    /* 0x28 */ PLAYER_AP_BOTTLE_CHATEAU,
+    /* 0x29 */ PLAYER_AP_BOTTLE_FAIRY,
+    /* 0x2A */ PLAYER_AP_MOON_TEAR,
+    /* 0x2B */ PLAYER_AP_DEED_LAND,
+    /* 0x2C */ PLAYER_AP_ROOM_KEY,
+    /* 0x2D */ PLAYER_AP_LETTER_KAFEI,
+    /* 0x2E */ PLAYER_AP_BEAN,
+    /* 0x2F */ PLAYER_AP_DEED_SWAMP,
+    /* 0x30 */ PLAYER_AP_DEED_MOUNTAIN,
+    /* 0x31 */ PLAYER_AP_DEED_OCEAN,
+    /* 0x33 */ PLAYER_AP_LETTER_MAMA = 0x33,
+    /* 0x36 */ PLAYER_AP_PENDANT_MEMORIES = 0x36,
+    /* 0x3A */ PLAYER_AP_MASK_TRUTH = 0x3A,
```

Comment:
are we sure we know these missing values are never used? should we just include `PLAYER_AP_UNK32` or something incase?

Reply:
I started to fill them in, but then `UNK11` seemed ambiguous on whether it was hex or decimal so I just left them empty. I'm fine putting them in if that's something that seems strange only to me